include/dlaf/common/index2d.h

```diff
@@ -24,6 +24,9 @@
 
 namespace dlaf {
 
+// TODO: Combine with below? What to call this?
+enum class TODOCoord { Row, Col, Full };
```

Comment:
```suggestion
enum class CommunicatorType { Row, Col, Full };
```
? Other suggestions for names?

And possibly
```
enum class CommunicatorType { Row = static_cast<int>(Coord::Row), Col = static_cast<int>(Coord::Col), Full };
```

Reply:
I've renamed it to `CommunicatorType` but leaving the `static_cast`s out for now since there's no immediate need for it. If we need to ensure they use the same values we can enforce that later.

Reply:
Marking this as resolved, but feel free to unresolve if you disagree with the current naming.